test/integration/coredump/test_coredump.py

```diff
@@ -38,7 +38,7 @@ def apport_crash_to_coredump(crash):
     if not core_pattern.startswith('|'):
         core_wildcard = core_pattern.strip().split('%')[0] + '*'
         if not os.path.isabs(core_wildcard):
-            core_wildcard = coredump_tmpdir / core_wildcard
+            core_wildcard = os.path.join(coredump_tmpdir.as_posix(), core_wildcard)
```

Comment:
What problem is fixed here?

Reply:
Previous version just didn't work, because core_wildcard is expected to be string later
```python
glob.glob(core_wildcard)
```
but it was not.

Thought it was me who wrote the previous version, I have no idea why it turned out like this.
The only version -- it was the last-minute-change. Tests where real coredump are generated are not the part of CI, neither general not full-ci (one can run them explicitly by specifying the proper env), so it seems I just haven't check them after that last-minute-change.

Reply:
As for me, it's a more easy and clear way to write `glob.glob(str(core_wildcard))`. Than such an odd expression when it's not quite clear at a glance exactly what he's doing.
The question was mostly why need `.as_posix()`?

Reply:
I think I got your point. However considering that:
- This code is changed in the next PR (#1059)
- This change doesn't relate directly to the issue

in order to avoid wasting time I dropped this change completely from this PR.